lib/landrush/cap/host/redhat/install_dnsmasq.rb

```diff
@@ -0,0 +1,14 @@
+module Landrush
+  module Cap
+    module Redhat
+      module InstallDnsmasq
+        class << self
+          def install_dnsmasq(_env)
+            system('sudo yum clean all > /dev/null 2>&1')
+            system('sudo yum install -y -q dnsmasq > /dev/null 2>&1')
+          end
```

Comment:
Vagrant does not provide host type check for Fedora out of the box. It does hardly for any OS. That's the reason why I had my own XYZHost classes.